docs_app/package.json

```diff
@@ -133,7 +133,7 @@
     "tree-kill": "^1.2.2",
     "ts-node": "^8.2.0",
     "tslint": "~6.1.0",
-    "typescript": "4.3.5",
+    "typescript": "^4.3.5",
```

Comment:
We probably don't want to do this. TypeScript has breaking changes between minor versions.

Reply:
Thanks @benlesh.  I updated it to 4.3.2.  On a side note, the rxjs library looks to be using ^4.2.2.